docs/src/test-reporter-api/class-reporter.md

```diff
@@ -37,15 +37,15 @@ module.exports = MyReporter;
 
 ```js tab=js-ts title="my-awesome-reporter.ts"
 import type {
-  Reporter, FullConfig, Suite, TestCase, TestResult, FullResult
+  Reporter, ReporterConfig, Suite, TestCase, TestResult, FullResult
 } from '@playwright/test/reporter';
 
 class MyReporter implements Reporter {
   constructor(options: { customOption?: string } = {}) {
     console.log(`my-awesome-reporter setup with customOption set to ${options.customOption}`);
   }
 
-  onBegin(config: FullConfig, suite: Suite) {
+  onBegin(config: ReporterConfig, suite: Suite) {
```

Comment:
Now this looks like it's a config of the reporter itself, rather than resolved test config.